ci/cluster-api/vsphere/templates/cluster.yaml

```diff
@@ -44,9 +44,9 @@ spec:
     spec:
       cloneMode: linkedClone
       datacenter: DATACENTERNAME
-      datastore: vsanDatastore
+      datastore: DATASTORENAME
```

Comment:
I suppose that the DATASTORE and FOLDERNAME are defined in ${WORKDIR}/utils/CLUSTERDEFAULTS-* ?

Reply:
Yes, we won't need to change any upstream code for job migration by this way.